src/styles/units/mixins.scss

```diff
@@ -0,0 +1,27 @@
+@mixin cardButton() {
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding